kubernetes/services/ignite_alcor.yaml

```diff
@@ -92,7 +92,7 @@ spec:
             - name: OPTION_LIBS
               value: ignite-kubernetes,ignite-rest-http
             - name: CONFIG_URI
-              value: https://raw.githubusercontent.com/kevin-zhonghao/alcor/xml/kubernetes/services/ignite_config.xml
+              value: http://10.213.43.169:8082/ignite_config.xml
```

Comment:
@pkommoju It seems not a good idea to put our internal public ip address on the public repository.

Reply:
I think we will have to find a better alternative soon but for now, this could be acceptable. Since you released three machines to me, I will use them to figure out the way to force ignite use the config file of our choice.

Reply:
10.213.43.169 is non-routable so security concern is not a big deal but others will have to edit the yaml file to point to some machine in their internal/external host.

We could use https://github.com/futurewei-cloud/alcor/blob/master/kubernetes/services/ignite_config.xml. This way it is under our control and publicly accessible.

Reply:
I am assuming that this file is also available in Alcor master, if that is the case, we can replace kevin's link with Alcor master link.